BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -23,6 +23,8 @@
 import ntpath
 from   CommonUtility import *
 from   IfwiUtility   import FLASH_MAP, FLASH_MAP_DESC, FIT_ENTRY, UCODE_HEADER
+from   SignUtility import *

```

Comment:
If a customer needs to use two separate utilities for production and non-production, he/she will have to modify/update all the tools.
Can SignUtility.py be included only from CommonUtility.py ?
And 'CommonUtility.py' should be the only interface for 'SignUtility' with SBL code.